packages/runtime/src/helpers/blobStorage.ts

```diff
@@ -0,0 +1,34 @@
+import type { Blobs } from '@netlify/blobs/dist/src/main'
```

Comment:
This and memoize won't end up being part of this PR as they're already in #2287.